Cesium.CodeGen.Tests/verified/CodeGenArrayTests.StaticStructArrayTest.verified.txt

```diff
@@ -0,0 +1 @@
+﻿emptyString
```

Comment:
Huh? What happened here? It yield no codegen results?

Reply:
Yes. I need to check only emitted classes. Maybe we need new type of tests.

Reply:
Don't worry, I'll handle this myself.